integration-tests/support/pages/tabs/PipelinerunsTabPage.ts

```diff
@@ -194,9 +194,13 @@ export class LogsTab {
   static downloadAllTaskLogs(allTaskLogs = true) {
     LogsTab.goToLogsTab();
     if (allTaskLogs) {
-      cy.contains('button', pipelinerunsTabPO.downloadAllTaskLogsButton).click();
+      cy.contains('button', pipelinerunsTabPO.downloadAllTaskLogsButton)
+        .should('have.attr', 'aria-disabled', 'false')
```

Comment:
would this work with just
```typescript
.should('be.enabled')
```
instead of attribute check?

Reply:
Okay 👍 , updated.